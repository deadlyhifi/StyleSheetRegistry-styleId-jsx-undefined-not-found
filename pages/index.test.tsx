import * as React from "react";
import { cleanup, render } from "react-testing-library";
import Home from ".";

afterEach(cleanup);

it("renders as expected", () => {
  const { container } = render(<Home />);

  expect(container).toMatchSnapshot();
});
