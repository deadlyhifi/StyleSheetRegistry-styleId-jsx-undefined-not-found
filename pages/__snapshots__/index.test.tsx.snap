// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div>
  <ul
    class="jsx-undefined"
  >
    <a
      class="jsx-undefined"
      data-qa="example1"
      href="https://www.example1.com"
      rel="noopener external"
      target="_blank"
      title="Example 1"
    >
      <li
        class="jsx-undefined"
      >
        <img
          alt="example1 logo"
          class="jsx-undefined"
          src="/static/icon.png"
        />
      </li>
    </a>
    <a
      class="jsx-undefined"
      data-qa="example2"
      href="https://www.example2.com"
      rel="noopener external"
      target="_blank"
      title="Example 2"
    >
      <li
        class="jsx-undefined"
      >
        <img
          alt="example2 logo"
          class="jsx-undefined"
          src="/static/icon.png"
        />
      </li>
    </a>
  </ul>
</div>
`;
